bring-your-own-datacenter/values.yaml

```diff
@@ -11,8 +11,10 @@ byodc:
     https: null
     "no": null
     ftp: null
+    all: null
   # configuration settings for the byodc
   config:
+    nodeSelector: {}

```

Comment:
Should we add a comment to this as well?

Reply:
Maybe a link to this page or so
https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector